browser/src/slideshow/PresenterConsole.js

```diff
@@ -382,18 +384,34 @@ class PresenterConsole {
 		this._presenter.getNavigator().onClick(e);
 	}
 
-	_onToolbarClick(e) {
-		let target = e.target;
-		let elem;
+	_getButton(target) {
+		while (target && target.localName !== 'button') {
+			target = target.parentElement;
+		}
+
+		return target;
+	}
+
+	_onToolbarMouseDown(e) {
+		let target = this._getButton(e.target);
 		if (!target) {
 			return;
 		}
+		target.style.transform = 'translateY(5px)';
```

Comment:
Hello @hcvcastro code looks good but can we add a method for adding transform animation ?

Like a method `xyz(Element , add/remove:boolean)` this can help to add animation to other elements as well without repeating the code

Any thoughts ?

Reply:
Oh i see it works for all toolbar button. Please ignore my previous comment.